src/pages/mesh/advanced/logging.md

```diff
@@ -53,3 +53,17 @@ aio api-mesh:log-get-bulk --startTime 2024-08-27T21:31:39Z --endTime 2024-08-27T
 The time format is `YYYY-MM-DDTHH:MM:SSZ`. You must convert your local time to UTC.
 
 For more information see [`aio api-mesh:log-get-bulk`](./index.md#aio-api-meshlog-get-bulk).
+
+## Export bulk logs with a relative time range
+
+You can use the `--past` flag to specify the number of minutes in the past to get logs. The maximum value is `30`.
+
+```bash
+aio api-mesh:log-get-bulk --past 30 --filename mesh_logs.csv
```

Comment:
@jhadobe, the `--past `flag should include the time unit explicitly( eg: 30mins, 30minutes as a single token) than just the numeric value